build.sbt

```diff
@@ -416,9 +420,9 @@ lazy val benchmarks = project
 
 val validateCommands = List(
   "clean",
-  "compile",
   "scalafix --check",
   "scalafmtCheckAll",
```

Comment:
Do we still need these two if compile would trigger format already?

Reply:
Because there is no guarantee you would compile the code locally.